contents/docs/advanced/proxy.mdx

```diff
@@ -56,4 +57,4 @@ If you want to use an alternative reverse proxy that we have not documented, it
 - route: e.yourdomain.com/*
   reverse_proxy: https://us.i.posthog.com/*
   host_header: us.i.posthog.com
-```
\ No newline at end of file
+```
```

Comment:
This adds an extra `\n` to the end of the file, couldn't get rid of it in the Gitlab's editor.